packages/now-build-utils/tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "outDir": "./dist",
     "types": ["node", "jest"],
     "strict": true,
-    "target": "esnext"
+    "target": "esnext",
+    "resolveJsonModule": true
```

Comment:
I've had weird bugs from enabling this feature so you might want to use `JSON.parse(fs.readFile())` instead.

See #3315 for more info.

Reply:
Done 👍